io.ossia.score.yml

```diff
@@ -0,0 +1,333 @@
+id: io.ossia.score
+runtime: org.kde.Platform
+runtime-version: '6.7'
+sdk: org.kde.Sdk
+command: ossia-score
+finish-args:
+  # BLE support
+  - --allow=bluetooth
+
+  # Access user's data and media libraries, plug-ins etc
+  - --filesystem=host
+
+  # GPU rendering support
+  - --device=dri
+
+  # Raw input support
+  - --device=input
+
+  # Required for USB & MIDI devices support as well as JACK (/dev/shm) and raw GPIO I/O
+  - --device=all
```

Comment:
MIDI devices are exposed through the pulseaudio socket permission. The comment is incorrect.

Reply:
also this doesn't enable `/dev/shm` and JACK will never work that way. The platform provides `pw-jack` instead.

Reply:
oh, I read multiple comments that --device=all would enable /dev/shm. well :)

for MIDI : are also Alsa RawMidi devices available that way (basically raw access to a /dev, needed for some features not supported by the ALSA Sequencer API and necessary for hardware that supports more than classic MIDI 1 speed and large sysex: https://github.com/thestk/rtmidi/issues/214), as well as pipewire MIDI devices (needed for MIDI BLE support) ?

Reply:
and - is raw JACK still possible ? even with pipewire 1.2.0 I still encounter cases where it's worth running JACK1 in terms of raw performance - or even raw ALSA PCM support (this is really necessary for instance when the software runs on e.g. Raspberry Pi Zero where just the pw daemon running and doing nothing is already using too much resources).

Reply:
wouldn't it be possible to just bind-mount host libjack if it exists? (ossia loads jack through dlopen to enable support for systems that do not provide JACK at all so it does not care about a specific libjack version)

Reply:
No. It is not.